Documentation/Migration/Index.rst

```diff
@@ -174,6 +200,121 @@ of typing a long :bash:`docker run...` or :bash:`podman run...` command:
+We can now check via google if there is another link to book Extbasefluid. We found this site `https://docs.typo3.org/m/typo3/book-extbasefluid/10.4/en-us/`
+We can find the hint: `This manual is no longer being maintained for TYPO3 versions 11.5 and above.`. This tells us that the developers stopped to update the
+documentation. We can for example link to the last existing version. Which is 10.4. To do this we have to change the :file:`guides.xml`. Search for the
+
+.. code-block:: xml
+
+    <inventory id="t3extbasebook"
+           url="https://docs.typo3.org/m/typo3/book-extbasefluid/11.5/en-us/"
+    />
+
+and change `11.5` to `10.4`. Generate files again.
+
+
+..  _migrate-inventory-link-with-key-not-found:
+
+Inventory link with key ... not found
+"""""""""""""""""""""""""""""""""""""
+
+.. code-block:: text
+
+    [2024-03-13T12:26:40.940930+00:00] app.WARNING: Inventory link with key "h2document:rest-common-pitfalls"
+    (rest-common-pitfalls) not found.  {"rst-file":"GeneratedExtension/Index","type":"ref","targetRef
+
+We see already that we have to go to file :file:`GeneratedExtension/Index` in the directory "Documentation".
+In there we have to delete the line which contains
+
+.. code-block:: text
+
+    * :ref:`h2document:rest-common-pitfalls`
+
+..  _migrate-nested-php-domain-components-not-supported:
+
+Nested PHP domain components (php:class, php:interface, php:enum etc) are not supported
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+.. code-block:: text
+
+    [2024-03-25T13:26:11.600367+00:00] app.WARNING: Nested PHP domain components
+    (php:class, php:interface, php:enum etc) are not supported.
+    Found php:\Vendor\MyExtension\Interfaces\RequireJsModuleInterface inside
+    \Vendor\MyExtension\Interfaces\AnotherImportantInterface {"rst-file":"Developer/Index.rst"} []
+
+The file :file:`Index.rst` in Documentation/Developer has a wrong indentation. A class must not belong to another class.
+Here is the wrong rst code.
+
+.. code-block:: rst
+
+    ..  php:class:: AnotherImportantInterface
+
+    Used for ...
+
+    ..  php:class:: RequireJsModuleInterface
+
+    Widgets implementing this interface will add the provided RequireJS modules.
+    Those modules will be loaded in dashboard view if the widget is added at least once.
+
+Additionally regarding the name it has to be interfaces and not classes.
+`..  php:class:: ExampleInterface` has to be changed to `..  php:interface:: ExampleInterface`.
+
+
+..  _migrate-reference-could-not-be-resolved:
+
+Reference sitehandling-addinglanguages could not be resolved in LocalizedContent/Index {"rst-file":"LocalizedContent/Index"} []
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+The next step is to visit the old site, e.g. `https://docs.typo3.org/m/typo3/guide-frontendlocalization/main/en-us/LocalizedContent/Index.html`.
```

Comment:
Why **old** site?

Reply:
Because you can see if the refernce was still working in the old rendering?

Reply:
By "old" I meant the old links when using the Sphinx Rendering. As "new" I would refer to the PHP-based rendering.

Reply:
But how can a user visit an "old site" when it is already rendered with guides?

As you see, I haven't understand it, so maybe our readers also don't.